### ISSUE TITLE
Feature Request: Move between places of Edit

### ISSUE THREAD

**Issue by brodders61** [spyder-ide/spyder#3192]
Hi guys,

this probably in wrong place but always wanted in spyder a way to get back to place of last edit.

My work involves complex data structures represented in diverse places; I need to check these as I work. So after I've found something and worked out what I need to do, I'm often lost! Where in the file was I? If only I could get to the place of last edit easily!

Thus suggest:

Alt <left arrow> step back to prior edit position
Alt <right arrow> step forward to next edit position

ono. Thank you.
:)

**Comment by goanpeca** [comment]
@brodders61 thanks for reporting.

People have asked for bookmarks before and this request seems to fit in that description. If this locations you are referring to are more or less fixed you could use special comments  that appear in the outline explorer

```python

# --- Special comment that will appear in the outline explorer
# Normal comment
```

**Comment by brodders61** [comment]
:)

Thats great!

Thanks Gonzalo!
________________________________

**Comment by goanpeca** [comment]
No prob!

**Comment by brodders61** [comment]
Gonzalo,

This is a useful feature, but I still get lost as there are places without comment marks.

Yes, it is a help - but then the edit history knows where the locations are. I was hoping to just bind some keystrokes to the locations...

:( ??
Steve B
________________________________

**Comment by goanpeca** [comment]
@brodders61 sure, I was just giving an intermediate aid. 

Will mark it as enhancement and looking for help :-p

**Comment by brodders61** [comment]
Hi Gonzalo,

Have been trying the new 'previous cursor position', 'next cursor position' in Spyder which I suggested - thanks for putting those in!

Be advised that on Windows PCs the key bindings chosen (Ctrl-Alt-L and -R) are intercepted by hardware (on Intel and Nvidia based machines) as 'rotate primary screen' and are not passed through to the application stack. Yep, I've got a PC here and all my icons got repositioned as well (to fit the new orientation). I discovered Ctrl-Alt-Up resets the orientation - phew!

Changing the key bindings does not seem to help... yes the unwanted screen orientation problem goes away, but the action performed is 'step current file tab' left and right. Not sure why.

But re-assigning 'last edit location; to Alt-F1 worked and that is useful!

:)
all good fun,
Steve B
________________________________
